recipes/openal/all/conanfile.py

```diff
@@ -60,7 +60,7 @@ def package(self):
 
     def package_info(self):
         if self.settings.os == "Windows":
-            self.cpp_info.libs = ["OpenAL32", 'winmm']
+            self.cpp_info.libs = ["OpenAL32", 'winmm', 'OLE32', 'Shell32']
```

Comment:
```suggestion
            self.cpp_info.libs = ["OpenAL32", "winmm"]
            self.cpp_info.system_libs = ["OLE32", "Shell32"]
```

Reply:
What about "winmm"? It should be in "system_libs" too I suppose - it is NOT provided by this package.

Reply:
that's what I did in https://github.com/conan-io/conan-center-index/pull/1132/files